addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -340,6 +340,8 @@ export class WebsitePreview extends Component {
                 // Forward clicks to close backend client action's navbar
                 // dropdowns.
                 this.iframe.el.dispatchEvent(new MouseEvent('click', ev));
+            } else {
+                ev.preventDefault();
```

Comment:
A comment about this would not be too much IMO :wink: (actually real question: is it required for something else than links?)

Reply:
> actually real question: is it required for something else than links?

Good question, I did not really think about what it affects, but I guess it might be justified with a "better safe than sorry" approach.
It might be worth reviewing it's usefulness on everything but I guess it's at least required for "old school forms" that use a `<button type="submit">` that also trigger navigation. Though there might be other use cases that I'm not thinking of.